com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/HDRP/Water/EvaluateWaterSimulationAdditionalDataNode.cs

```diff
@@ -0,0 +1,68 @@
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+    internal class EvaluateWaterSimulationAdditionalDataNode : IStandardNode
+    {
+        public static string Name => "EvaluateWaterSimulationAdditionalData";
+        public static int Version => 1;
+
+        public static FunctionDescriptor FunctionDescriptor => new(
+            Name,
+@"WaterAdditionalData waterAdditionalData;
+ZERO_INITIALIZE(WaterAdditionalData, waterAdditionalData);
+EvaluateWaterAdditionalData(UV0.xyz, BandsMutliplier, waterAdditionalData);
+SurfaceGradient = waterAdditionalData.surfaceGradient;
+LowFrequencySurfaceGradient = waterAdditionalData.lowFrequencySurfaceGradient;
+SurfaceFoam = waterAdditionalData.surfaceFoam;
+DeepFoam = waterAdditionalData.deepFoam;
+",
+            new ParameterDescriptor[]
+            {
+                new ParameterDescriptor("UV0", TYPE.Vec4, Usage.Local, REF.UV0),
+                new ParameterDescriptor("BandsMutliplier", TYPE.Vec4, Usage.In, new float[] { 1f, 1f, 1f, 1f}),
```

Comment:
You've got a typo in the name of this parameter.  Switch the t and the L.  Make this change in the ParameterUIDescriptor as well.

Reply:
great catch! Also a typo in sg1